src/Raven.Client/Documents/Indexes/IndexDefinition.cs

```diff
@@ -17,6 +18,8 @@ namespace Raven.Client.Documents.Indexes
     /// </summary>
     public class IndexDefinition
     {
+        private static readonly Regex _regex = new Regex("\r*\n", RegexOptions.Compiled);
```

Comment:
```suggestion
        private static readonly Regex _regex = new Regex("\r?\n", RegexOptions.Compiled);
```

That is a more correct way to do that, no?

Reply:
In theory yes, in practice... maybe :) Pushed.